deploy/jN/includes/CSSLint.js

```diff
@@ -25,12 +87,12 @@ function getCSSLint(){
 	return CSSLint;
 }
 
-var settings = GlobalSettings.get("CSSLint") || {};
+var settings = GlobalSettings.set("CSSLint") || {};
 
 var CSSLintMenuCfg = {
 	text:"CSS Lint",
 	oninitpopup:function(){
-		this.verifyMenuItem.disabled  = !/\.css/i.test(currentView.files[currentView.file]);
+		this.verifyMenuItem.disabled  = !/\.(css|php|html)/i.test(currentView.files[currentView.file]);
```

Comment:
does the csslint work also on php and html files properly? Maybe it is worth to make it part of settings or control it via (un)check in the menu?